### ISSUE TITLE
name 'unicode' is not defined, the second

### ISSUE THREAD

**Issue by donpatrice** [mdavidsaver/p4p#30]
Python 3.7.2
p4p 3.1.2

Running...
`$ python -m p4p.server.cli my:pv:name=int`
I get
```
Traceback (most recent call last):
  File "/home/patrik/.pyenv/versions/3.7.2/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/patrik/.pyenv/versions/3.7.2/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/patrik/.local/share/virtualenvs/cryvisil-daq-939BQ-ad/lib/python3.7/site-packages/p4p/server/cli.py", line 23, in <module>
    'enum': nt.NTEnum().wrap(0),
  File "/home/patrik/.local/share/virtualenvs/cryvisil-daq-939BQ-ad/lib/python3.7/site-packages/p4p/nt/enum.py", line 66, in wrap
    self.assign(V, value)
  File "/home/patrik/.local/share/virtualenvs/cryvisil-daq-939BQ-ad/lib/python3.7/site-packages/p4p/nt/enum.py", line 86, in assign
    if isinstance(py, (bytes, unicode)):
NameError: name 'unicode' is not defined
```
Related to https://github.com/mdavidsaver/p4p/issues/6

**Comment by mdavidsaver** [comment]
It's strange that this passes my unit tests with python 3.6.  The fix is clear enough though.

**Comment by mdavidsaver** [comment]
Actually, can you run the tests as well with py3.7?  `python -m nose p4p`

**Comment by donpatrice** [comment]
Yes the tests pass with py3.7.

**Comment by mdavidsaver** [comment]
Fixed with 8f8a7af9048eaf77ff87116bd02dafda707083f1.  Also added test coverage.

**Comment by mdavidsaver** [comment]
Released in 3.2.0
